### ISSUE TITLE
Link to other collective in view mode forwards to filelist on mobile chrome

### ISSUE THREAD

**Issue by mejo-** [nextcloud/collectives#408]
In GitLab by @susnux on Oct 25, 2022, 11:40

When clicking on a link to an other collective / collective page in the normal "view"-mode,
the selected collective is opened but then the user is directly forwarded to the filelist ( `https://SERVER/apps/files` ).
You can click the back button to get to the selected collective but this is extremely inconvenient.

This happens only on mobile (Android) Chrome, multiple users noticed this issue.
Server version is latest 24 (24.0.6).

**Comment by mejo-** [comment]
@susnux are you still able to reproduce this issue? I tried to reproduce with Chromium on Android but failed the following way:

1. Create a collective, copy the link to it
2. Create a second collective
3. Create a new page in second collective, put link to first collective in the page
4. Change to view mode and click the link

With all browsers I tested (Firefox Desktop, Chromium Desktop, Chromium Android) the linked collective was opened as expected. I didn't spot a redirection to the files app.

Did I miss something?
